lib/features/theory/models/theory_data_model.dart

```diff
@@ -0,0 +1,24 @@
+class TheoryTopic {
+  final int count;
+  final String topic;
+  final String theory;
+
+  TheoryTopic(this.count, this.topic, this.theory);
```

Comment:
Use named params
TheoryTopic(
{
   required this.count, 
   required this.topic,
   required this.theory,
}
)